src/libxc/hyb_gga_xc_wb97.c

```diff
@@ -15,7 +15,7 @@
  
 typedef struct { 
   double c_x[5], c_ss[5], c_ab[5]; 
-#if defined CUDA || defined CUDA_MPIV
```

Comment:
Why is cam_omega not used in CPU code?

Reply:
Oh it should! Thanks for pointing this out. I think we never detected this since meta GGAs are disabled.

Reply:
In CPU code, cam_omega is set by initialize function into params struct: https://github.com/merzlab/QUICK/blob/0e1ecf816bc65aa3567bc3f5a47786b0155562cc/src/libxc/hyb_gga_xc_wb97.c#L60-L104. In CUDA version, we just set them directly inside functional structs.